setup.py

```diff
@@ -27,7 +27,7 @@
         ],
     },
     tests_require=[
-        'mock',
+        'mock==1.0.1',
```

Comment:
Yeah, agreed, if there is a problem with the CCI setup, tweak the circle.yml and leave this tests_require alone